src/main/java/uk/nhs/ctp/entities/SupplierPartitioned.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.ctp.entities;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+import lombok.Data;
+
+@Data
+@MappedSuperclass
+public class SupplierPartitioned {
+
+  @Column(name = "supplierToken")
+  private String supplierToken;
```

Comment:
As with FHIR server, this is probably better described as an ID rather than a token (although purely semantics)